tests/migration_test.go

```diff
@@ -425,6 +493,68 @@ var _ = Describe("Migrations", func() {
 				By("Waiting for VMI to disappear")
 				tests.WaitForVirtualMachineToDisappearWithTimeout(vmi, 120)
 			})
+			It("should be able successfully cancel a migration", func() {
+
+				vmi := tests.NewRandomVMIWithEphemeralDisk(tests.ContainerDiskFor(tests.ContainerDiskFedora))
+				vmi.Spec.Domain.Resources.Requests[k8sv1.ResourceMemory] = resource.MustParse("1Gi")
+				tests.AddUserData(vmi, "cloud-init", fmt.Sprintf(`#!/bin/bash
+					echo "fedora" |passwd fedora --stdin
+					yum install -y stress qemu-guest-agent
```

Comment:
Uh, that makes the test online-only. We already serve the qemu-guest-agent binary from an http server inside the test environment. Can we do the same with `stress`?

Reply:
Ok, but can you please push me to the right direction? where do we do it?

Reply:
I see that such tests somehow already slipped into the repo. Not your task to fix it. We can fix it in a separate pr. You can continue with yum from my side in this PR.